guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md

```diff
@@ -3,626 +3,2178 @@ group: release-notes
+
+
+* **Declarative schema** simplifies installation and upgrade procedures for Magento and extensions. Declarative schema reduce the need for many database scripts, eliminating the need to maintain these scripts. And here's a big advantage: This features enables Magento to roll out database schema changes in patch releases (not currently possible). This feature supports split and shared database structures and database structure validation. 
+
+* **GraphQL API** provides an alternative to REST and SOAP web APIs for frontend development. See [GraphQL Developer Guide]({{site.baseurl}}/guides/v2.3/graphql/index.html) for more information about Magento's implementation of this data query language. 
+
+* **Asynchronous Web APIs** allow any previous Magento REST APIs to be called asynchronously. This community-contributed feature includes separate status APIs that have been created to check the status of each request. Developers can now use the asynchronous APIs  in conjunction with queues that have also been migrated to Magento Open Source.  See [Asynchronous web endpoints](https://devdocs.magento.com/guides/v2.3/rest/asynchronous-web-endpoints.html) for more information. 
+
+
+* **Bulk Web APIs**  allow all existing REST APIs to accept payloads with multiple entities. These community-contributed bulk APIs support more efficient and scalable implementations that eliminate round-trip network overhead. Like asynchronous APIs, bulk web APIs can be used in conjunction with queues that have also been migrated to Magento Open Source. [See Bulk endpoints](https://devdocs.magento.com/guides/v2.3/rest/bulk-endpoints.html) for more information. 
+
+* **Updates to Magento's tech stack (including upgraded PHP support to maintain PCI compliance)** include upgrades to Redis, MySQL, Elasticsearch, compatibility with PHP 7.2.x. 
+
+
+
+### Substantial security enhancements
+
+	* Over 30 security fixes to core Magento code.
+
+    * Cache flush ACL provides granular access to cache management settings to prevent accidental changes that could potentially affect system performance. This ACL also lets merchants control which administrative users can clear site caches. 
+
+    * 2FA/CAPTCHA protects the Admin panel against against stolen passwords and protects stores against bots.
+
+### Core bundled extension enhancements
+
+#### Amazon Payments
+
+* Added branding to the Amazon Pay config section in admin
+
+* Improved extension architecture and performance
+
+
+#### dotmailer
+
+* 	dotmailer now supports the Magento Commerce split database mode. 
+
+
+#### Klarna Payments
+
+* Added descriptive text to the Refund API call  
+
+* Added a link to the Klarna merchant portal  
+
+* Show a detailed Klarna error message in the Magento Admin where needed  
```

Comment:
To be consistent w. previous bulleted points, I believe it's possible to make it: "Added a detailed Klarna message in the Magento admin where needed.